master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,152 @@
+---
+title: AWS
+---
+
+Calico is designed to provide high performance massively scalable virtual
```

Comment:
We should mention that Calico lets you bypass the 50-node limit that exists when using VPC routes (e.g Flannel AWS VPC mode)